waves/wallet/src/wallet/decompose_transaction.rs

```diff
@@ -67,8 +67,27 @@ pub async fn decompose_transaction(
         })
         .collect::<Result<Vec<_>, UnblindError>>())?;
 
-    Ok(TransactionElements {
```

Comment:
Why did you opt for removing `TransactionElements` in the favor of one amount being negative and one positive?